synapse/static/index.html

```diff
@@ -0,0 +1,26 @@
+<html>
+   <head>
+       <title>Synapse is running</title>
+       <style>
+           body {
+               width: 30em;
+               margin: 0 auto;
+               font-family: Tahoma, Verdana, Arial, sans-serif;
+           }
+           h1 {
+               text-align: center;
+           }
+       </style>
+   </head>
+   <body>
+       <h1>Synapse is running</h1>
+       <p>Congratulations!</p>
+       <p>Your Synapse server is listening on this port and is ready for messages.</p>
+       <p>To use this matrix-server you'll need a client to use this server - e.g. one of
```

Comment:
seems duplicative, """to use this server you need a client to use this server"""